release/src/Main.hs

```diff
@@ -40,6 +40,13 @@ import qualified SdkVersion
 scalaVersions :: [T.Text]
 scalaVersions = ["2.12.12", "2.13.3"]
 
+-- Build-time dependencies to exclude
+depsToExclude :: T.Text
+depsToExclude = T.intercalate " + " [
+   "//compiler/scenario-service/protos:scenario_service_java_proto",
+   "//compiler/repl-service/protos:repl_service_java_proto",
+   "//daml-script/runner:script-runner-lib-ce"]
```

Comment:
I'd like @garyverhaegen-da, @moritzkiefer-da, or someone equally familiar with this script to review this part.

Reply:
The workaround is fine with me and we had it before. The problem is that the check that we don’t forget to include transitive dependencies is broken. It should be checking the what the jar depends on. What it actually checks is all build time dependencies via `bazel query`. As soon as you depend on a DAR somehow you pull in the compiler which depends on those 3 libs.

I think trying to fix the `bazel query` is relatively hopeless, we should do the check based on the pom files or something like that but that’s a larger change.

I think we should copy parts of my explanation here into a comment but apart from that, I’m fine with this.

Reply:
I added a comment